model_compiler_2/setup.py

```diff
@@ -65,7 +65,8 @@ def run(self):
             'mypy',
             'pylint',
             'pytest-cov',
-            'pytest-xdist'
+            'pytest-xdist',
+            'torch'
```

Comment:
It seems that `torch` is only used for generating ONNX models, we can use `tf2onnx` to do this so that we can eliminate this dependency completely.